travis-results-3.txt

```diff
@@ -0,0 +1,205 @@
+---------------------------------------
+>>> Working on metering book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with metering book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on web_console book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with web_console book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on monitoring book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with monitoring book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on authentication book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with authentication book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on logging book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 178: callout list item index: expected 7, got 6
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 179: callout list item index: expected 8, got 7
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 180: callout list item index: expected 9, got 8
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 181: callout list item index: expected 10, got 9
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 182: callout list item index: expected 11, got 10
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 183: callout list item index: expected 12, got 11
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 183: no callout found for <12>
+>>> Finished with logging book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on scalability_and_performance book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with scalability_and_performance book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on cli_reference book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with cli_reference book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on backup_and_restore book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+asciidoctor: WARNING: includes/dr-recover-expired-control-plane-certs.adoc: line 36: unterminated listing block
+[31m[1mUnknown ID or title "dr-restoring-cluster-state", used as an internal cross reference[0m
+[31m[1m>>> Validation of book backup_and_restore in openshift-enterprise failed <<<[0m
+>>> Finished with backup_and_restore book in openshift-enterprise <<<
+---------------------------------------
```

Comment:
Weird set of issues here, like the internal cross references and the validation issue. Not sure why that happened, but it was distracting to the warning that is important.

This warning is for the closing `----` in a code block in one of the test modules.